assets/scss/app.scss

```diff
@@ -163,4 +163,22 @@ ul {
 .notice {
   margin-top: 0.5em;
   margin-bottom: 2em;
+
+#tenant-additional-details-group h2 {
```

Comment:
This is needed because by default the h2 is very large and not appropriate for this page.